### DISCUSSION TITLE
customizing title block in website format

### DISCUSSION THREAD

**Discussion by graemeblair** [quarto-dev/quarto-cli#3725] [Q&A] [answered]
Hi there - I'm trying to customize the title block to add two new YAML options ("Translated by" and "Revised by") in addition to "Author" in a page within a website project. 

I found your discussion for the journal type (https://github.com/quarto-dev/quarto-cli/discussions/3162), but this links to the partial [title-block.html ](https://github.com/quarto-dev/quarto-cli/blob/56888bb94984fd93f16edb42dbe5564c6dc65eaa/src/resources/formats/html/pandoc/title-block.html) which doesn't seem to be used for the web site type (for example, it doesn't include affiliations). 

Is there a partial file I can adapt that's used for web site pages or do I need to use a lua filter or something else? Thank you for any suggestions!

**Comment by mcanouil** [comment, marked as answer]
This is the right partial for the title-block.
Affiliation is in `title-metadata.html` which is included in title-block.html (https://github.com/quarto-dev/quarto-cli/blob/main/src/resources/formats/html/templates/title-block.html)
